README.md

```diff
@@ -1,12 +1,24 @@
 # xblock-group-project-v2
 
-
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
 ## Features
 
+**Team Evaluations**
+ 
+Workgroup members can provide anonymous feedback to each other.
+
+**Grading**
+ 
+Activities are either peer-graded or staff-graded:
+
+* *Staff grading* - staff members with appropriate roles are asked to provide grade for the workgroup.
+* *Peer grading* - students are asked to evaluate other workgroups' work.
+* *Staff-grading fallback* - if some of the students fail to provide grades to another workgroup, staff members can 
+  interfere and provide missing grades.
+
```

Comment:
These two concepts are core selling points of GWv2 IMO, and notoriously were problematic for me, so I decided that if someone glaces through it, he should know the distinction.